components/loading/LoadingBar.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <v-app ref="loaderContainer" class="v-app">
+    <v-progress-linear
+      :active="loading"
+      :indeterminate="indeterminate"
+      :height="4"
+      :bottom="true"
+      :rounded="true"
+      :color="barColor"
+      :value="barValue"
+    ></v-progress-linear>
+  </v-app>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+let timeout: number;
```

Comment:
This could probably use a better name and should, if possible, be part of the component (But I'm not sure if this component is destroyed and re-mounted or just hidden).

Reply:
@MrTimscampi Is hidden. I thought setting this here "globally" made more sense, as it's not a variable that you will manipulate externally (from the Vue.js debugger or other components), you will only use it in the progress bar component itself. It isn't relevant for anything else but the progress bar either.

Agree with you about the name though, I'll make it clearer.

Reply:
You generally want to keep variables either in components, or in a Vuex store :smiley: 

Defining variables just outside of a component like this, to be then used inside of it, is pretty bad practice :wink: